.github/workflows/pull_requests.yml

```diff
@@ -24,13 +24,13 @@ jobs:
         with:
           files: |
             **/*.md
-      - uses: ruby/setup-ruby@v1
+      - uses: actions/setup-node@v3
```

Comment:
I suspect the semantics of ruby/setup-ruby aren't valid for actions/setup-node.

https://github.com/actions/setup-node#usage

I am a github actions novice but I suspect we want something like this.
```
- uses: actions/setup-node@v3
  if: steps.changed-files.outputs.any_changed == 'true'
  with:
    node-version: 18
    cache: 'npm'
```

Reply:
Oh, right, missed that 🙈